cmd/runhcs/kill.go

```diff
@@ -49,6 +68,78 @@ signal to the init process of the "ubuntu01" container:
 			return err
 		}
 		defer p.Close()
-		return p.Kill() // BUGBUG: should be Signal
+
+		if signalsSupported {
+			opts := guestrequest.SignalProcessOptions{
+				Signal: signal,
+			}
+			return p.Signal(opts)
+		}
+
+		// Legacy signal issue a kill
+		return p.Kill()
 	},
 }
+
+func validateSigstr(sigstr string, signalsSupported bool, isLcow bool) (int, error) {
```

Comment:
This is really dumb using a string, but I guess out of your control?

Reply:
Yea. Oh well